### PR TITLE
OSDOCS-4670:updates Networking metrics for 4.12 RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-4670](https://issues.redhat.com//browse/OSDOCS-4670): updates enhancements to networking metrics in RNs for 4.12

Version(s):
4.12

Issue:
[Jira](https://issues.redhat.com/browse/OSDOCS-4670)

Link to docs preview:
[Preview](https://53643--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-networking)

QE review:
- [x] QE has approved this change.

Additional information:
@martinkennelly PTAL

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -170,9 +170,23 @@ A new import value, `importMode`, has been added to the `importPolicy` parameter
 [id="ocp-4-12-networking-metrics"]
 ==== Enhancements to networking metrics
 
-The following metric is now available for OVN-Kubernetes CNI network provider clusters:
+The following metric is now available for OVN-Kubernetes network plug-in:
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:173]:
```suggestion
The following metrics are now available for OVN-Kubernetes network plug-in:
```

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-12-release-notes.adoc]: Nice catch, thanks.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -170,9 +170,23 @@ A new import value, `importMode`, has been added to the `importPolicy` parameter
 [id="ocp-4-12-networking-metrics"]
 ==== Enhancements to networking metrics
 
-The following metric is now available for OVN-Kubernetes CNI network provider clusters:
+The following metrics are now available for OVN-Kubernetes network plug-in:
```
Comment (jboxman) [inline release_notes/ocp-4-12-release-notes.adoc:173]:
```suggestion
The following metrics are now available for the OVN-Kubernetes network plug-in:
```

Comment (jboxman-rh) [inline release_notes/ocp-4-12-release-notes.adoc:173]:
```suggestion
The following metrics are now available for the OVN-Kubernetes network plug-in:
```

**Pull request merged**